contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -124,6 +155,24 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
   /// exists.
   error CapabilityAlreadyExists();
 
+  /// @notice This error is thrown when trying to add a node to a DON where
+  /// the node does not support the capability
+  /// @param donId The ID of the DON
+  /// @param nodeP2PId The P2P ID of the node
+  /// @param capabilityId The ID of the capability
+  error NodeDoesNotSupportDONCapability(uint32 donId, bytes32 nodeP2PId, bytes32 capabilityId);
```

Comment:
```suggestion
  error NodeDoesNotSupportCapability(bytes32 nodeP2PId, bytes32 capabilityId);
```
I think DON is not relevant in this error. WDYT?

Reply:
updated